elections/2023-SC-EU/README.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Knative Steering Committee end user Seat"
+linkTitle: "Steering Committee end user Seat"
+weight: 30
+type: "docs"
+---
+
+# 2023 Knative Steering Committee end user Seat
```

Comment:
I know previous year has been named as 2022 EU seat, but maybe we should stop it and just say 2024 EU seat?